endoreg_db/management/commands/fix_missing_patient_data.py

```
@@ -0,0 +1,172 @@
+                                fixed_count += 1
+                        except Exception as e:
+                            self.stdout.write(
+                                self.style.ERROR(f"Failed to update SensitiveMeta for video {video.uuid}: {e}")
+                            )
+                    else:
+                        fixed_count += 1
+        
+        # Summary
+        self.stdout.write("\n" + "="*50)
+        if dry_run:
+            self.stdout.write(self.style.SUCCESS(f"DRY RUN SUMMARY:"))
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)